packages/@react-spectrum/dropzone/intl/en-US.json

```
@@ -0,0 +1,3 @@
+{
+  "message": "Drop file to replace"
```

Mega nit for clarity
```suggestion
  "replaceMessage": "Drop file to replace"
```